Makefile

```diff
@@ -8,11 +8,27 @@ ifeq ($(AUTH_FILE),)
 AUTH_FILE = /etc/serviceaccount/service-account.json
 endif
 
+ifdef PULL_PULL_SHA
```

Comment:
This is working as expected with the prow setup. I think it will be good to remove this part once we completely delete the prow related components, otherwise this will mean more changes to the `README.md`. what do you think?

Reply:
ok, lets leave it and hope we don't forget to remove it after the switch.